services/workflows-service/src/collection-flow/collection-flow.service.ts

```diff
@@ -229,6 +235,51 @@ export class CollectionFlowService {
     return flowData;
   }
 
+  private async __persistFileUrlsToDocuments(
```

Comment:
It's hard to understand what this function does, while called `persist` it doesn't seem to persist anything, the final return array also fixed on `http` provider which I don't get